codegen/sdk-codegen/build.gradle.kts

```diff
@@ -208,13 +193,6 @@ task("stageSdks") {
                 from("${it.outputDir}")
                 into("${it.sourcesDir}")
                 exclude("Package.swift")
-                exclude("*Tests")
-            }
-            logger.info("copying ${it.outputDir} tests to ${it.testsDir}")
-            copy {
-                from("${it.outputDir}")
-                into("${it.testsDir}")
-                include("*Tests/**")
```

Comment:
The extra copy step is no longer needed because tests are included in the package structure with code, like a standard Swift package.